.travis.yml

```diff
@@ -15,3 +15,5 @@ script:
 - go test -i -race ./...
 - go test -v -race ./...
 - ./scripts/cov.sh TRAVIS
+after_script:
+- if [ "$TRAVIS_GO_VERSION" = "1.5" ] && [ "$BUILD_GOOS" = "linux" ] && [ "$TRAVIS_TAG" != "" ]; then ./scripts/cross_compile.sh; ghr --username derekcollison --token $GITHUB_TOKEN --replace $TRAVIS_TAG pkg/; fi
```

Comment:
btw I think this one here would have to be `ghr --username nats-io` so that the built artifacts are put under `nats-io/gnatsd`